### PR TITLE
WOOC-486 : Show oney TAEG and fees on checkout page

### PR DESCRIPTION
**Pull request by GhDj**

### CODE REVIEW

File: src/Front/PayplugOney/Requests/OneyBase.php [hunk 1]
```diff
@@ -182,4 +184,12 @@ public function setOney($oney)
 		$this->oney = $oney;
 	}
 
+	public function addOneyJs() {
```
Comment (PPmmesquita) [inline src/Front/PayplugOney/Requests/OneyBase.php:187]: do we need this payplug-oney.js?

File: src/Gateway/PayplugGatewayOney3x.php [hunk 2]
```diff
@@ -96,6 +96,7 @@ public function get_icon()
             $total_price = floatval(WC()->cart->total);
             $this->oney_response = $this->api->simulate_oney_payment($total_price, 'with_fees');
             $currency = get_woocommerce_currency_symbol(get_option('woocommerce_currency'));
+			$total_price_oney = $total_price + $this->oney_response['x3_with_fees']['nominal_annual_percentage_rate'];
```
Comment (PPmmesquita) [inline src/Gateway/PayplugGatewayOney3x.php:99]: this won't affect the product page values?

Comment (GhDj) [reply src/Gateway/PayplugGatewayOney3x.php:99]: no, this is only about showing the installments on the checkout page, not the popup

File: src/Front/PayplugOney/Requests/OneyBase.php (hunk 1 above)
Comment (GhDj) [reply src/Front/PayplugOney/Requests/OneyBase.php:187]: no we don't I am going to delete this, I used before changing the requirments on the US, will push a roll back

**Review by PPmmesquita — APPROVED** [review]

**Pull request merged**